static/src/js/containers/AuthCallbackContainer/__tests__/AuthCallbackContainer.test.js

```diff
@@ -154,7 +154,41 @@ describe('AuthCallbackContainer component', () => {
     expect(setSpy).toBeCalledTimes(0)
 
     expect(window.location.replace.mock.calls.length).toBe(1)
-    expect(window.location.replace.mock.calls[0]).toEqual(['/'])
+    expect(window.location.replace.mock.calls[0]).toEqual(['/not-found'])
+  })
+
+  test('does not follow the redirect if the redirect param is not relative to earthdata-search', () => {
+    const setSpy = jest.spyOn(tinyCookie, 'set')
+    delete window.location
+    window.location = { replace: jest.fn() }
+
+    setup({
+      location: {
+        search: '?redirect=https://evil.com'
+      }
+    })
+
+    expect(setSpy).toBeCalledTimes(0)
+
+    expect(window.location.replace.mock.calls.length).toBe(1)
+    expect(window.location.replace.mock.calls[0]).toEqual(['/not-found'])
+  })
+
+  test('does not follow the eddRedirect it is not a valid earthdata-download redirect', () => {
```

Comment:
I don't know that this test is really very helpful. its very similar to the test below it and running though the same code. I added it for explicitness when it comes to what solution this PR is trying to do but, I'll pull it out if people deem to totally redundant